source.txt

```diff
@@ -506,6 +506,11 @@ Table of Contents
     SHOULD connect to the given provider account, as defined in Section
     10.
 
+    If the `remotestorage` field exists in the URL fragment, the application
+    SHOULD ignore any other parameters such as `access_token` or `state`, to
+    ensure compatibility with servers that implement older versions of this
+    specification.
+
```

Comment:
I think the rest of the spec uses single quotes to mark code and special terms. Looks like you were thinking in Markdown mode.